app/assets/javascripts/app/form-validation.js

```diff
@@ -21,7 +20,6 @@ const validate = {
   },
 
   addEvents() {
-    this.form.addEventListener('change', e => validateField(e.target));
```

Comment:
Removing this will make it so that client side validation only happens once the submit button is pushed rather than real time feedback when someone removes focus from an input they were on.